modules/face/samples/samplewriteconfigfile.cpp

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
header comment

also wondering if we need software for this ... since people could simply edit an xml, but lets keep it for now.